ZenPacks/zenoss/LinuxMonitor/zenpack.yaml

```diff
@@ -187,6 +187,45 @@ device_classes: !ZenPackSpec
                         datapoints:
                             status:
                                 rrdmin: 0
+                    iostats:
+                        type: COMMAND
+                        commandTemplate: '/usr/bin/sudo /usr/bin/cat /proc/diskstats'
```

Comment:
Looks like file /proc/diskstats is readable for all users. So do we need sudo here at all?